hopsworks-IT/src/test/ruby/spec/factories/host.rb

```diff
@@ -17,4 +17,5 @@ class Host < ActiveRecord::Base
   def self.table_name
     "hosts"
   end
+  attribute :num_gpus, ActiveRecord::Type::Integer.new
```

Comment:
@kai-chi What's this used for?

Reply:
this is caused by the fact that ruby interprets TINYINT as a boolean and not as a number. SO here: https://stackoverflow.com/questions/39971023/rails-sees-mysql-tinyint1-as-a-boolean-but-i-want-it-to-be-a-number

Reply:
Ok can you add a comment then?